dev/io.openliberty.jakarta.jsf.3.0/bnd.bnd

```diff
@@ -0,0 +1,41 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
```

Comment:
Looking at the com.ibm.websphere.javaee.jsf.2.3 feature I see we disable instrumentation we should probably do the same here unless I'm looking at something incorrectly. Please verify for me.